src/pages/resultsView/ResultsViewPage.tsx

```diff
@@ -5,6 +5,7 @@ import URL from 'url';
 import { inject, observer } from 'mobx-react';
 import { action, computed, observable, reaction, runInAction } from 'mobx';
 import { ResultsViewPageStore } from './ResultsViewPageStore';
+import { PatientViewPageStore } from '../../pages/patientView/clinicalInformation/PatientViewPageStore';
```

Comment:
importing `PatientViewStore` in `ResultsViewPage` does not seem right to me

Reply:
doesnt appear to even be used so it'll be removed